src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/EnvelopeEventProcessorTest.java

```diff
@@ -158,12 +172,18 @@ public void should_dead_letter_the_message_when_notification_sending_fails() thr
             eq(DEAD_LETTER_REASON_PROCESSING_ERROR),
             eq(exceptionMessage)
         );
+        verify(appInsights).trackDeadLetteredMessage(
+            someMessage,
+            "envelopes",
```

Comment:
Worth referencing the queue name through a constant. Just to consider, up to you.